<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>78</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>78</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>78</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Passgen</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>6</y>
      <width>85</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinSymbols">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>40</y>
      <width>52</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkNums">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>42</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Numbers</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkSpec">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>42</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>#$&amp;*)(+/</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
